catalog/camel-catalog/src/generated/resources/org/apache/camel/catalog/docs/milo-client-component.adoc

```diff
@@ -228,6 +229,24 @@ Input parameters are taken from the body:
 * If the body is a `Variant`, then it will be wrapped in a `Variant[]` array
 * Otherwise the body will be converted into a `Variant` and wrapped in an array of `Variant[]`
 
+=== Read Values from Nodes
+
+The component provide a producer to read values from multiple opc-ua nodes. The Node-IDs will be defined in the header `CamelMilo.nodeIds` as list of strings. (see <<nodeid, Node-ID>> for the ID format).
+
+Example:
+```java
+from("direct:start")
+    .setHeader("CamelMilo.nodeIds", constant(Arrays.asList("nsu=urn:org:apache:camel;s=myitem1")))
```

Comment:
The header key in this doc was not updated

Reply:
omg. Sorry for that. I fixed it.